CMakeLists.txt

```diff
@@ -755,6 +755,17 @@ if(UMF_FORMAT_CODE_STYLE)
     endif()
 endif()
 
+find_package(Python3)
+if(Python3_FOUND)
+    message(STATUS "Adding 'docs' target")
```

Comment:
Maybe add: (...) for creating a documentation.

Reply:
done.